frontend/src/components/PomodoroButtons.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { useSelector, useDispatch } from "react-redux"
+import { timer } from "../reducers/timer"
+import { ButtonsContainer, InnerButtonContainer, TimerButton, TimerIcon, BigIcon } from "styled-components/Styling"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import {
+  faRedo,
+  faPlayCircle,
+  faTimes,
+  faPauseCircle,
+} from "@fortawesome/free-solid-svg-icons"
+
+const PomodoroButtons = ({ 
+  setSeconds,
+  setMinutes,
+  workMinutes,
+  setSecondsLeft,
+  totalSeconds,
+  work}) => {
+  const dispatch = useDispatch()
+
+  const isRunning = useSelector((state) => state.timer.isRunning)
+  const activatedButton = useSelector((state) => state.timer.items._id)
+
+  const ReplayIcon = <FontAwesomeIcon icon={faRedo} />
+  const PlayIcon = <FontAwesomeIcon icon={faPlayCircle} />
+  const StopIcon = <FontAwesomeIcon icon={faTimes} />
+  const PauseIcon = <FontAwesomeIcon icon={faPauseCircle} />
+
+  const buttonBackgroundColor = work ? "var(--red)" : "var(--blue)"
+  const iconColor = work ? "var(--lightRed)" : "var(--lightBlue)"
+
+  return (
+    <>
+      <ButtonsContainer buttonBackgroundColor={buttonBackgroundColor}>
+        <InnerButtonContainer>
+          <TimerButton
+            disabled={!activatedButton}
+            onClick={() => {
+              dispatch(timer.actions.setisRunning(false))
+              setSeconds(0)
+              setMinutes(workMinutes)
+              setSecondsLeft(totalSeconds)
+            }}
```

Comment:
This is too much of logic to pass directly in JSX - would be great if you could create separate function